content/assignments/memory-allocator/utils/printf.c

```diff
@@ -0,0 +1,920 @@
+///////////////////////////////////////////////////////////////////////////////

```

Comment:
TBH I don't think it's a problem to solve this one.

Reply:
Will do.